### PR TITLE
Fix build issues on Ubuntu 17.10 against latest FFmpeg snapshot

### PR DESCRIPTION
**Pull request by carun**

### CODE REVIEW

File: dub.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
             "dependencies":{
                 "dcv:core": "*",
                 "imageformats": "~>6.1.0",
-                "ffmpeg-d": "~>2.5.1"
+                "ffmpeg-d": "~master"
```
Comment (ljubobratovicrelja) [inline dub.json]: Why not link to [3.1.0](https://code.dlang.org/packages/ffmpeg-d)? It should contain changes that are currently at master afaik. If that is the case, please change `master` --> `3.1.0`.

**Review by ljubobratovicrelja — CHANGES_REQUESTED** [review]
Hej @carun thanks for this! Beside the comment on ffmpeg-d version in the dub.json, can you address the issues that are occurring in travis? Can you tell us what is actually going on there?

File: dub.json (hunk 1 above)
Comment (carun) [reply dub.json]: Hi @ljubobratovicrelja, I didn't know which version to use, so I used `master`. Will update to use `3.1.0`.

I'm testing with LDC 1.6.0 and DMD 2.077.1 on my local. Regarding travis failures, do we need to support all the old versions? They are almost > 8 months old. Please confirm.

Comment (ljubobratovicrelja) [reply dub.json]: > Regarding travis failures, do we need to support all the old versions? They are almost > 8 months old

Fair point. In that case, would you be so kind and edit .travis.yml to test newer versions of both compilers? I hope DMD >= 2.0.74.1 and LDC >= 1.4.0 (same front-end as dmd 2.0.74.1) and  will pass alright. Don't worry if you'd prefer not to do it, just say the word. I'll deal with it right after the merge.

Comment (carun) [reply dub.json]: Sure, let me try it.

Comment (carun) [reply dub.json]: Not sure why travis build fails even after the change. @ljubobratovicrelja could you tell me more?

File: dub.json [hunk 2]
```diff
@@ -65,7 +65,7 @@
             "dependencies":{
                 "dcv:core": "*",
                 "imageformats": "~>6.1.0",
-                "ffmpeg-d": "~master"
+                "ffmpeg-d": "~3.1.0"
```
Comment (ljubobratovicrelja) [inline dub.json]: Maybe try `"ffmpeg-d": "~>3.1.0"` (missing the `>` sign)?

**Review by ljubobratovicrelja — APPROVED** [review]

**Comment by ljubobratovicrelja** [comment]
Nice! Can we merge?

**Comment by carun** [comment]
If you say so! 👍

**Pull request merged**